packages/dialtone-vue3/components/popover/popover.test.js

```diff
@@ -92,7 +92,7 @@ describe.skip('DtPopover Tests', () => {
         expect(popoverWindow.exists()).toBe(true);
       });
 
-      it('should render the main content', () => {
+      it.skip('should render the main content', () => {
```

Comment:
I think perhaps the same problem you're having with your hovercard tests, is the same reason these tests are skipped.

Reply:
Might be, although in vue 3 they are not skipped. Might have something to do with the fact that the hovercard adds another wrapper to all this?